### ISSUE TITLE
Overclocking doesn't work (version 3.3 Wheezy)

### ISSUE THREAD

**Issue by phaseIV** [RetroPie/RetroPie-Setup#1141]
Overclocking doesn't work with the Wheezy version (/boot is empty) while the Jesse variant gives no problems with it. Am i missing something?

**Comment by joolswills** [comment]
Thanks.

Looks like something got mangled when building the images.

I'll fix fstab and re-roll a 3.3.1

In the meantime you can add

```
/dev/mmcblk0p1  /boot           vfat    defaults          0       2
/dev/mmcblk0p2  /               ext4    defaults,noatime  0       1
```

to /etc/fstab

**Comment by phaseIV** [comment]
Thank you. This fixes the overclocking.

**Comment by joolswills** [comment]
There is a new image available now (3.3.1)
